docs/src/main/sphinx/connector/accumulo.rst

```diff
@@ -24,6 +24,15 @@ JAR file to Accumulo's ``lib/ext`` directory on each TabletServer node.
 
     # TabletServer should pick up new JAR files in ext directory, but may require restart
 
+Requirements
+------------
+
+To connect to Accumulo, you need:
+
+* Accumulo version 1.7.4 or higher.
+* Network access from the Trino coordinator and workers to the Accumulo
+  Zookeeper server. Port 2181 is the default port.
```

Comment:
Assuming this means 2181 is the port for the Zookeeper server, but would like that made explicit. "...is the default port for the Zookeeper server"

Reply:
Repeating would be repetitive. CC @mosabua